README.md

```diff
@@ -1,6 +1,6 @@
 # addons-scanner-utils
 
-[![Build Status](https://travis-ci.com/mozilla/addons-scanner-utils.svg?branch=master)](https://travis-ci.com/mozilla/addons-scanner-utils) [![codecov](https://codecov.io/gh/mozilla/addons-scanner-utils/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla/addons-scanner-utils) [![npm version](https://badge.fury.io/js/addons-scanner-utils.svg)](https://badge.fury.io/js/addons-scanner-utils)
+[![codecov](https://codecov.io/gh/mozilla/addons-scanner-utils/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla/addons-scanner-utils) [![npm version](https://badge.fury.io/js/addons-scanner-utils.svg)](https://badge.fury.io/js/addons-scanner-utils)
```

Comment:
Adding a circle badge might be nice? It's handy at a glance to know that everything is green.

Reply:
Yep, thanks for pointing me to the docs. I thought we could generate a badge from the settings (which is how I get badges for private repos).